src/pages/edge/adobe-journey-optimizer-decisioning/review-simulate.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Adobe Experience Platform Assurance Review & Simulate
+description: This is a feature of the Adobe Experience Platform Assurance which allows the users to review the content sent to devices for each request and simulate different experiences on the same devices in real time.
+keywords:
+- Adobe Experience Platform Assurance
+- Review & Simulate Feature Introduction
+---
+
+# Adobe Experience Platform Assurance Review & Simulate
+
+This is a feature of the Adobe Experience Platform Assurance which allows the users to review the content sent to devices for each request and simulate different experiences on the same devices in real time.
+
+## Review Requests and Results
+
+1. [Setup and connect](./assurance-for-decisioning.md) ot an Assurance session.
+2. To review the personlization requests and results navigate to ‘Review & Simulate’ tab.
```

Comment:
"Review the personalization requests and results by navigating to the Review and Simulate tab."

Reply:
I think the tab is called `Requests`, not `Review & Simulate`